pdns/protobuf.cc

```diff
@@ -97,6 +97,57 @@ void DNSProtoBufMessage::setEDNSSubnet(const Netmask& subnet, uint8_t mask)
 #endif /* HAVE_PROTOBUF */
 }
 
+// ----------------------------------------------------------------------------
+// GCA - Seth Ornstein - 5/30/2017 - extra protobuf information
+// ----------------------------------------------------------------------------
+
+void DNSProtoBufMessage::addTags(const std::string& strLabel, const std::string& strValue)
+{
```

Comment:
I think we should move `RecProtoBufMessage::setPolicyTags()` and `RecProtoBufMessage::getPolicyTags()` here instead.